src/søknad/steg/4-barnasbosted/BarnetsBostedLagtTil.tsx

```diff
@@ -185,7 +185,7 @@ const BarnetsBostedLagtTil: React.FC<Props> = ({
             <BodyShort>{forelder.borAnnenForelderISammeHus.verdi}</BodyShort>
           </div>
         ) : null}
-        {forelder?.boddSammenFør?.svarid ? (
+        {forelder?.boddSammenFør?.verdi ? (
```

Comment:
Dette blir vel feil?
Skal vel vise spørsmål og svar selv om det er besvart "Nei" her? Samme for linje 199

Reply:
Godt sett! Sjekker nå om forelder?.boddSammenFør?.verdi har verdi.